mmv1/products/alloydb/api.yaml

```diff
@@ -0,0 +1,286 @@
+        path: "error"
+        message: "message"
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: "clusterId"
+        required: true
+        input: true
+        url_param_only: true
+        description: |
+          The ID of the alloydb cluster.
+      - !ruby/object:Api::Type::String
+        name: "location"
+        input: true
+        url_param_only: true
+        description: |
+          The location where the alloydb cluster should reside.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: "name"
+        output: true
+        description: |
+          The name of the cluster resource.
+      - !ruby/object:Api::Type::Time
+        name: "createTime"
+        description: |
+          Time the Cluster was created in UTC.
+        output: true
+      - !ruby/object:Api::Type::Time
+        name: "updateTime"
+        description: |
+          Time the Cluster was updated in UTC.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: "uid"
+        output: true
+        description: |
+          The system-generated UID of the resource.
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: "labels"
+        description: "User-defined labels for the alloydb cluster."
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: "annotations"
+        description: "Annotations to allow client tools to store small amount of arbitrary data. This is distinct from labels."
+      - !ruby/object:Api::Type::String
+        name: "network"
+        required: true
+        description: |
+          The relative resource name of the VPC network on which the instance can be accessed. It is specified in the following form:
+
+          "projects/{projectNumber}/global/networks/{network_id}".
+      - !ruby/object:Api::Type::String
+        name: "state"
+        output: true
+        description: |
+          The current state of the alloydb cluster.
+      - !ruby/object:Api::Type::String
+        name: "selfLink"
+        output: true
+        description: |
+          The selfLink of the alloydb cluster.
+      - !ruby/object:Api::Type::String
+        name: "displayName"
+        description: |
+          User-settable and human-readable display name for the Cluster.
+      - !ruby/object:Api::Type::String
+        name: "databaseVersion"
+        output: true
+        description: |
+          The database engine major version. This is an output-only field and it's populated at the Cluster creation time. This field cannot be changed after cluster creation.
+      - !ruby/object:Api::Type::Boolean
+        name: "reconciling"
+        output: true
+        description: |
+          Set to true if the current state of Cluster does not match the user's intended state, and the service is actively updating the resource to reconcile them. This can happen due to user-triggered updates or system actions like failover or maintenance.
```

Comment:
This has implications that the resource design might get complicated. For example if we deploy/provision the resource what should happen if the state is reconciling.. do we wait for a good state before completing deployment. How are we qualifying if this resource has been /provisioned/. If other resources are dependent on it. Does it need to be in a certain state?

Reply:
Also, not sure how useful exposing this field will be to the end user since it's dynamic.

Reply:
+1 fields that can change out of band may not be accurate in the state.

Reply:
removed